backend/segmentation/models/microtubule/segment_mt.py

```
@@ -73,6 +77,76 @@ def _normalize(img: np.ndarray) -> np.ndarray:
+    which is why the pipeline ran on Linux/macOS but not Windows.
+
+    Remapping the concrete classes to their ``Pure`` counterparts keeps the
+    values intact and inspectable everywhere; ``PurePosixPath`` constructs on
+    any OS because it does pure lexical path handling with no filesystem
+    access — which is all these two values were ever going to get.
+    """
+
+    _PURE_EQUIVALENT = {
+        ("pathlib", "PosixPath"): pathlib.PurePosixPath,
+        ("pathlib", "WindowsPath"): pathlib.PureWindowsPath,
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)